### DISCUSSION TITLE
Should I make a issue about bugs that also present in Firefox?

### DISCUSSION THREAD

**Discussion by NeonEDuck** [zen-browser/desktop#1171] [Q&A] [answered]
Just found a bug relate to "Open previous windows and tabs" in General setting, where if the private window is open, reopening the normal window won't open the tabs on startup.

It seems to forget what tabs was opened or closed, since trying to hit `Ctrl`+`Alt`+`T` to restore the tabs also won't work.

I have checked and found out that Firefox also have the same issue!
So I was wondering if I should make a issue for this bug or not.

**Comment by nocanoa** [comment, marked as answer]
If the bug is reproducible in Firefox Vanilla, then pls don't make a issue.
That should be reported to FF, so Zen gets the bug fix from upstream.
